proxy/tls.go

```diff
@@ -124,35 +125,131 @@ func newTLSListenerConfig(conf *config.Config) (*tls.Config, error) {
 
 	cfg.VerifyPeerCertificate = func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
```

Comment:
The verification logic got quite big, maybe it would be better to move it to a separate function.

Reply:
Moved out into a separate function: https://github.com/grepplabs/kafka-proxy/pull/65/files#diff-2ee690fbbffb319d2291746fea4d8eeeR142.